src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -57,8 +57,13 @@
     public static final String COMMENT = "#";
     public static final String POSITIONAL_ARGUMENTS_NAME = "Positional Argument";
 
-    // Extension for collection argument list files
-    static final String COLLECTION_LIST_FILE_EXTENSION = ".args";
+    /**
+     * Extensions recognized for collection argument file expansion.
+     */
+    static final String COLLECTION_LIST_FILE_EXTENSIONS_LIST = ".list";
+    static final String COLLECTION_LIST_FILE_EXTENSIONS_ARGS = ".args";
```

Comment:
I think that it might be better to add the class for configuration at this stage (see https://github.com/broadinstitute/barclay/pull/95), because each of this changes is generating a breaking change. I think that the configuration is a breaking change but flexible enough to avoid the change  of the collection file extension here and keep it only in GATK.

I will open a new PR with that solution to have a clean branch instead of an outdated PR that get lost because its purpose was changed.

Reply:
Taking #95 would require a major version bump (by semantic versioning, which I'd like to follow as much as possible), which we just did, and which I'd like to avoid until we can bundle any other such changes together. This PR introduces only a minor change in runtime behavior (no source or binary incompatibility), and the runtime behavior change has a pretty easy workaround (to change the file extension being used).

It think we've accumulated enough feature requests that involve customization that we need a even more general mechanism than the config class as in #95. Instead, I think we should take this change as is, and implement the scheme outlined in (https://github.com/broadinstitute/barclay/issues/127) is more sustainable in the long run.

Reply:
Sounds reasonable. Looking forward to the next version bump with #127 implemented (and maybe we can add the #68 (although it will require to clean up the branch, which is quite outdated).